### PR TITLE
merge 2020.06

### PR DESCRIPTION
**Pull request by kkrebs**

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 60949eb00ed49af6c91e1a3f4d619ff20057acbb! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/2763-pull_request-496

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 61c8dace93b45ce1ded175208689df9ee7db5e6f! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/2769-pull_request-496

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with d7f7986911a84eacd058d095da9b1c89f724bb9c! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/2796-pull_request-496

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5809127&bid=19016940)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5809127&bid=19016940)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Complexity increasing per file
==============================
- mir-module/src/main/resources/META-INF/resources/js/ckeditor-config.js  1
- mir-module/src/main/java/org/mycore/mir/editor/MIREditorUtils.java  1
- mir-module/src/main/java/org/mycore/mir/editor/MIRUnescapeResolver.java  5
- mir-module/src/main/java/org/mycore/mir/editor/MIRPostProcessor.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5809127&bid=19016940)

File: mir-module/src/main/resources/META-INF/resources/js/ckeditor-config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CKEDITOR.editorConfig = function( config ) {
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/ckeditor-config.js:1]: Codacy found an issue: ['CKEDITOR' is not defined.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5809127)

**Pull request merged**